xbmc/playlists/PlayList.h

```diff
@@ -14,6 +14,11 @@
 #include <string>
 #include <vector>
 
+#define PLAYLIST_NONE -1
+#define PLAYLIST_MUSIC 0
+#define PLAYLIST_VIDEO 1
+#define PLAYLIST_PICTURE 2
```

Comment:
You can use `constexpr` for type safety.

Reply:
done